### PR TITLE
refactor: Curated List - Vue Composition API

### LINKED ISSUE #3772: Refactor: Curated List - Vue Composition API

### PR DESCRIPTION
**Pull request by preschian**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

### What's new?

- [x] PR closes #3772
- [x] WIP, branching from https://github.com/kodadot/nft-gallery/pull/3771
- [ ] Requires deployment <rubick/magick_issue>
- [ ] <brief_description_of_what_I've_added>

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR

### Optional

- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=DY4SQF2iD456tH89aQtz5wv1EV3BbSW8wKKuMcwbmXaj1pM)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

**Comment by petersopko** [comment]
wfm 🚀

**Review by petersopko — APPROVED** [review]
✅

**Review by kkukelka — CHANGES_REQUESTED** [review]
2 tiny things 😅

File: pages/snek/create.vue [hunk 1]
```diff
@@ -4,10 +4,10 @@
     <b-tabs v-model="activeTab" destroy-on-hide expanded>
       <b-tab-item v-for="x in components" :key="x" :label="x">
         <component
-          :showExplainerText="showExplainerText"
           :is="x"
-          @navigateToCreateNftTab="switchToNft"
-          v-if="components[activeTab] === x" />
+          v-if="components[activeTab] === x"
```
[the remark this answers is not in the document]
Comment (preschian) [reply pages/snek/create.vue:8]: sure, that would be nice. will update it 👍

actually, these changes are from the main branch, it was triggered with lint-staged on my local

**Review by kkukelka — APPROVED** [review]
all good now

**Comment by kkukelka** [comment]
@preschian whats the payment range for this, i cannot tell from the linked tickets

**Pull request merged**